automation/test-linux.sh

```diff
@@ -7,6 +7,11 @@ namespace="kubevirt"
 ocenv="OC"
 k8senv="K8s"
 
+if [ "$TARGET" == "centos6" ] && [ "$(uname -m)" == "s390x" ]; then
+  echo "CentOS 6 does not support s390x."
+  exit 1
```

Comment:
This will always fail the e2e tests if run on s390X.
Wouldn't be better to just print out warning and exit with 0?

Reply:
It is intentional for the CentOS 6 test to fail when run on s390x, as this Linux distribution, according to my investigation in osinfo-db, does not provide images for this architecture.